5333/src/main/cpp/Robot5333.cpp

```diff
@@ -23,8 +23,8 @@ void Robot::AutonomousPeriodic() {}
 
 void Robot::TeleopInit() {}
 void Robot::TeleopPeriodic() {
-  double leftSpeed = -xbox->GetY(xbox->kLeftHand);
-  double rightSpeed = -xbox->GetY(xbox->kRightHand);
+  double leftSpeed = -joy->GetY() + joy->GetZ();
```

Comment:
Slight Problem with this joy logic in terms of when the robot is moving forwards at y = +-1 and turning at z = +- 1 (i.e. the sum is greater than 1 and is truncated), but it works well enough.

Reply:
See #15 for an issue and further development